NMS-10133: Missing information in downtime model docs (#6915)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/service-assurance/downtime-model.adoc

<downtime interval="30000" begin="0" end="300000" /><!-- 30s, 0, 5m --> <1>
<downtime interval="300000" begin="300000" end="43200000" /><!-- 5m, 5m, 12h --> <2>
<downtime interval="600000" begin="43200000" end="432000000" /><!-- 10m, 12h, 5d --> <3>
----

<1> From 0 seconds after an outage is detected until 5 minutes, set the polling interval to 30 seconds.
<2> After 5 minutes of an ongoing outage until 12 hours, set the polling interval to 5 minutes.
<3> After 12 hours of an ongoing outage until 5 days, set the polling interval to 10 minutes.

Note that after five days of an ongoing outage, the service will be polled only once per hour.

> Services are not deleted during this time.

Reviewer: Let's just not talk about deletion at all.
```suggestion
```

> //what status do services have? A and N? What does that stand for?

Reviewer: Services statues below.  `A` means managed and will be scheduled for polling.  Anything else is a "do not poll"
    MANAGED = "A"
    UNMANAGED = "U"
    DELETED = "D"
    FORCE_UNMANAGED = "F"
    NOT_MONITORED = "N"
    RESCAN_TO_RESUME = "R"
    RESCAN_TO_SUSPEND = "S"
    REMOTELY_MONITORED = "X"

> // does editing a model affect current outages or only new ones?

Reviewer: The downtime model doesn't affect outages - it affects the scheduling of poller tasks.  It's affected by outages (and doesn't know the difference between new/existing)
```suggestion
```

<downtime interval="30000" begin="0" end="300000" /><!-- 30s, 0, 5m --> <1>
<downtime interval="300000" begin="300000" end="43200000" /><!-- 5m, 5m, 12h --> <2>
<downtime interval="600000" begin="43200000" end="432000000" /><!-- 10m, 12h, 5d --> <3>
----

<1> From 0 seconds after an outage is detected until 5 minutes, set the polling interval to 30 seconds.
<2> After 5 minutes of an ongoing outage until 12 hours, set the polling interval to 5 minutes.
<3> After 12 hours of an ongoing outage until 5 days, set the polling interval to 10 minutes.

Note that after five days of an ongoing outage, the service will be polled only once per hour.

> Any edits to the downtime model take effect immediately with existing and new outages.

Reviewer: ```suggestion
Any edits to the downtime model take effect after restarting the pollerd daemon, and apply to both existing and new outages.
```

For example, if an existing outage is being polled every 30 seconds for five minutes, but you change the downtime model during that period to a 45-second interval, the outage will be polled every 45 seconds for the remainder of the five minutes.

> == Service statuses
> The following list describes service statuses.
> Only MANAGED (A) are scheduled for polling.
> The others are "do not poll".

Reviewer: Not relevant in this document
```suggestion
```

> * MANAGED = "A"
> * UNMANAGED = "U"
> * DELETED = "D"
> * FORCE_UNMANAGED = "F"
> * NOT_MONITORED = "N"
> * RESCAN_TO_RESUME = "R"
> * RESCAN_TO_SUSPEND = "S"
> * REMOTELY_MONITORED = "X"

Reviewer: not relevant here.  most of these are internal statuses that the user does not get to set or see.
```suggestion
```

Note that after five days of an ongoing outage, the service will be polled only once per hour.

Any edits to the downtime model take effect after restarting the pollerd daemon, and apply to both existing and new outages.
For example, if an existing outage is being polled every 30 seconds for five minutes, but you change the downtime model during that period to a 45-second interval, the outage will be polled every 45 seconds for the remainder of the five minutes.

> Pollerd identifies the services as either up or down.

Reviewer: This applies to pollerd services in general, and is not specific to the downtime model.
